src/hints/patricia.rs

```diff
@@ -0,0 +1,43 @@
+use std::collections::HashMap;
+
+use cairo_vm::hint_processor::builtin_hint_processor::hint_utils::{get_integer_from_var_name, insert_value_into_ap};
+use cairo_vm::hint_processor::hint_processor_definition::HintReference;
+use cairo_vm::serde::deserialize_program::ApTracking;
+use cairo_vm::types::exec_scope::ExecutionScopes;
+use cairo_vm::vm::errors::hint_errors::HintError;
+use cairo_vm::vm::vm_core::VirtualMachine;
+use cairo_vm::Felt252;
+use indoc::indoc;
+
+use crate::hints::types::DescentMap;
+use crate::hints::vars;
+
+pub const SET_AP_TO_NOT_DESCEND: &str = indoc! {r#"
```

Comment:
`not` seems like a misnomer here since it's setting always setting ito to 0

Reply:
Will rename it.